rclrs/package.xml

```diff
@@ -7,49 +7,12 @@
   <maintainer email="esteve@apache.org">Esteve Fernandez</maintainer>
   <license>Apache License 2.0</license>
 
-  <buildtool_depend>ament_cmake_ros</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
   <buildtool_depend>ament_cmake_export_crates</buildtool_depend>
-  <buildtool_depend>rclrs_common</buildtool_depend>
 
-  <buildtool_export_depend>ament_cmake_ros</buildtool_export_depend>
-  <buildtool_export_depend>ament_cmake_export_crates</buildtool_export_depend>
-  <buildtool_export_depend>rclrs_common</buildtool_export_depend>
-
-  <build_depend>builtin_interfaces</build_depend>
-  <build_depend>rcl_interfaces</build_depend>
-  <build_depend>rcl</build_depend>
```

Comment:
Even if you don't link rcl and the rest of the libraries via CMake, you still need to mark them as a dependency to let colcon know that they must be available at build time.

Reply:
Resolved in 3f3a205